src/supertux/console.cpp

```diff
@@ -384,7 +385,7 @@ Console::autocomplete()
   // depending on number of hits, show matches or autocomplete
   if (cmds.empty())
   {
-    m_buffer.addLines("No known command starts with \"" + prefix + "\"");
+    m_buffer.addLines(_("No known command starts with \"") + prefix + _("\""));
```

Comment:
Please use boost::format for this, as seen here:
https://github.com/SuperTux/supertux/blob/ed17b8153a084a1d34845414a0bbc4bf7c669ff8/src/supertux/menu/profile_menu.cpp#L37

(otherwise, translators will have to translate " which is silly).